src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -754,6 +754,16 @@ private String key() {
         @Override public String getUrl() throws IOException {
             return run().getUrl();
         }
+        @Override public TaskListener getListener() throws IOException {
+            StreamBuildListener l = run().listener;
+            if (l != null) {
+                return l;
+            } else {
+                // Seems to happen at least once during resume, but anyway TryRepeatedly will call this method again, rather than caching the result.
+                LOGGER.log(Level.FINE, "No listener yet for {0}", this);
```

Comment:
In practice I have only seen it printed once per build; the `listener` is set during `onLoad`, just apparently slightly later than when the pickles start dehydrating.